sys/androidmedia/gstamcvideodechybris.c

```diff
@@ -1876,7 +1881,10 @@ gst_amc_video_dec_handle_frame (GstVideoDecoder * decoder,
       switch (idx) {
         case INFO_TRY_AGAIN_LATER:
           GST_DEBUG_OBJECT (self, "Dequeueing input buffer timed out");
-          goto dequeue_error;
+          if (++num_touts >= max_touts)
```

Comment:
Please add a comment saying something like "Try to dequeue up to num_touts times before giving up on trying to decode the frame."